Dream-Home-PEABOT/FEE_DreamHome
# Om feature display questions component

### What does this PR do?
- Creates Question component
- -Styles question component

### Where should the reviewer start?
- `src/components/Question/Question.css`
- `src/components/Questions/Question.tsx`

### How should this be manually tested?
- run `npm` start
- application should compile and the `question` component should render on the DOM

### Any background context you want to provide?

### What are the relevant tickets?
closes #33
### Questions:

### Screenshots (if appropriate)
![Screen Shot 2021-01-07 at 9 27 57 AM](https://user-images.githubusercontent.com/56229864/103917320-b0842c00-50ca-11eb-8d03-ab6bce83f019.png)
